modules/gltf/gltf_document.h

```diff
@@ -73,6 +73,7 @@ class GLTFDocument : public Resource {
 
 private:
 	const float BAKE_FPS = 30.0f;
+	int _compatibility_version = 2;
```

Comment:
The property seems to use `0,1` as a range, so this seems off?

Reply:
Oops, I forgot to change this.